app/src/main/res/layout/faq_list_fragment.xml

```diff
@@ -25,10 +25,12 @@
       android:id="@+id/faq_fragment_recycler_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/faq_list_margin_start"
```

Comment:
For `sw600dp` introduce a file `layout-sw600dp/faq_list_fragment` and instead of relying on the outer margins make the width of `recyclerview` fixed to `560dp`. This will help us make sure that UI works correctly even for a screen which has `600dp` width.

Reply:
Done, added new file